### ISSUE TITLE
Establish consistency checks to check if any issues exist in Redis/Postgres

### ISSUE THREAD

**Issue by tomerovadia** [tomerovadia/VoterHelpLine#49]

**Comment by tomerovadia** [comment]
@benweissmann If I write the code for this do you have the bandwidth to help me figure out how to have lambda run it daily (daily? hourly?)? I figure we can throw errors to Sentry if anything is off. I'm already curious to run this on the demo from yesterday and see if we catch any inconsistencies.

**Comment by benweissmann** [comment]
Yup, I can definitely get this running nightly on Lambda once you write the code.
